scripts/recover-funds.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs');
+const ethers = require('ethers');
+const path = require('path');
+const loadWallet = require('../lib/http/utils/load-wallet.js');
+
+const keysDirectory = process.env.COINSENCE_WALLETS_DIR || path.join(__dirname, '../', 'keys');
+const coinsenceWallet = process.env.COINSENCE_WALLET_ADDRESS;
+const password = process.env.PASSWORD || "coinsence";
+
+let ethProvider = new ethers.getDefaultProvider('rinkeby');
+
+fs.readdir(keysDirectory, function (err, accounts) {
+  if (err) {
+    return console.log('Error fetching wallets: ' + err);
+  }
+  else {
+    accounts.forEach(function (account) {
+      let accountId = path.basename(account, '.json');
+      
+      if(accountId != ".gitkeep") {
```

Comment:
hmmm. maybe we should just add some exception handling to catch errors that happen in `loadWallet()`? 
because there could be other non wallet files in that directory.